specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2023-05-01-preview/appconfiguration.json

```
@@ -0,0 +1,2079 @@
+      "properties": {
+        "items": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Label"
+          }
+        },
+        "@nextLink": {
+          "description": "The URI that can be used to request the next set of paged results.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SyncTokens": {
+      "name": "Sync-Token",
+      "in": "header",
+      "description": "Used to guarantee real-time consistency between requests.",
+      "type": "string"
+    },
+    "ApiVersion": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to be used with the HTTP request.",
+      "required": true,
+      "type": "string"
+    },
+    "Endpoint": {
+      "name": "endpoint",
+      "in": "path",
+      "description": "The endpoint of the App Configuration instance to send requests to.",
+      "required": true,
+      "type": "string",
+      "x-ms-skip-url-encoding": true,
+      "x-ms-parameter-location": "client"
+    },
+    "KeyValueFields": {
+      "name": "$Select",
+      "in": "query",
+      "description": "Used to select what fields are present in the returned resource(s).",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "key",
+          "label",
+          "content_type",
+          "value",
+          "last_modified",
+          "tags",
+          "locked",
+          "etag"
+        ],
+        "x-ms-enum": {
+          "name": "KeyValueFields",
+          "modelAsString": true
+        }
+      },
+      "x-ms-parameter-location": "method",
+      "collectionFormat": "csv"
+    },
+    "LabelFields": {
+      "name": "$Select",
+      "in": "query",
+      "description": "Used to select what fields are present in the returned resource(s).",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "name"
+        ],
+        "x-ms-enum": {
+          "name": "LabelFields",
+          "modelAsString": true
+        }
+      },
+      "x-ms-parameter-location": "method",
+      "collectionFormat": "csv"
+    },
+    "SnapshotFields": {
+      "name": "$Select",
+      "in": "query",
+      "description": "Used to select what fields are present in the returned resource(s).",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "name",
+          "status",
+          "filters",
+          "composition_type",
+          "created",
+          "expires",
+          "retention_period",
+          "size",
+          "items_count",
+          "tags",
+          "etag"
+        ],
+        "x-ms-enum": {
+          "name": "SnapshotFields",
+          "modelAsString": true
+        }
+      },
+      "x-ms-parameter-location": "method",
+      "collectionFormat": "csv"
+    },
+    "Status": {
+      "name": "Status",
```

Query parameter names should be lower camel case.
```suggestion
      "name": "status",
```

---

@avanigupta 

This looks like a change we should take in this version.